logtracer/tracing/tracer.py

```diff
@@ -119,15 +120,14 @@ def _extract_google_trace_headers_if_present(self, incoming_headers):
         if B3_TRACE_ID not in incoming_headers and GOOGLE_LOAD_BALANCER_TRACE_HEADERS in incoming_headers:
             incoming_headers = dict(incoming_headers)
             try:
-                traceid, spanid_with_params = incoming_headers.get(GOOGLE_LOAD_BALANCER_TRACE_HEADERS, '').split('/')
-                if ';' in spanid_with_params:
-                    spanid, params = spanid_with_params.split(';')
-                else:
-                    spanid = spanid_with_params
-                incoming_headers[B3_TRACE_ID] = traceid
-                incoming_headers[B3_SPAN_ID] = spanid
-            except ValueError:
+                match = re.search(_TRACE_CONTEXT_HEADER_RE, incoming_headers[GOOGLE_LOAD_BALANCER_TRACE_HEADERS])
+            except TypeError:
                 pass
+            else:
+                if match:
+                    incoming_headers[B3_TRACE_ID] = match.group(1)
+                    incoming_headers[B3_SPAN_ID] = match.group(3)
+                    # trace_options = match.group(5)
```

Comment:
is this commented out by accident?

Reply:
I left it in to illustrate how the trace options would be retrieved but they're currently not used at the mo